bindings/python/htcondor2/_cred_check.py

```diff
@@ -79,9 +79,11 @@ def error(self):
 
     @property
     def services(self):
+        # The last line should be :ad-attr:`OauthServicesNeeded`, but that
+        # job-ad attribute isn't documented yet.
         """
         The list of services that were requested, as a comma separated
         list.  This will be the same as the job ClassAd attribute
-        `OAuthServicesNeeded`.
+        ``OAuthServicesNeeded``.
```

Comment:
Can we  use :ad-attr: here, or no?

Reply:
I could, but `OAuthServicesNeeded` is presently undocumented:
```
WARNING: apis/python-bindings/api/version2/htcondor2/credcheck @ 1 | ClassAd Attribute 'OAuthServicesNeeded' not defined in any ClassAd Documentation files
```

Reply:
... which results in something that looks like a link but which doesn't go anywhere useful, which seems bad.